ggshield/cmd/hmsl/__init__.py

```diff
@@ -12,13 +12,25 @@
 from ggshield.cmd.hmsl.quota import quota_cmd
 
 
+class NaturalOrderGroup(click.Group):
```

Comment:
This class would be a good candidate for the new top-level `ggshield.utils.click` package. `ggshield.utils` is where we want to store code which can easily be reused in other projects (no business-logic, no dependency on py-gitguardian...)

Reply:
Indeed! I moved it!